llpc/CMakeLists.txt

```diff
@@ -384,6 +384,8 @@ endif()
 if(EXISTS ${XGL_SPVGEN_PATH})
     set(SPVGEN_ENABLE_WERROR ${LLPC_ENABLE_WERROR} CACHE BOOL "${PROJECT_NAME} override." FORCE)
     add_subdirectory(${XGL_SPVGEN_PATH} ${CMAKE_BINARY_DIR}/spvgen EXCLUDE_FROM_ALL)
+    # Propagate the spvgen build directory to the parent scope so that XGL tests can use it.
+    set(XGL_SPVGEN_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/spvgen PARENT_SCOPE)
```

Comment:
Sorry for the late comment, I didn’t notice this earlier. I think you need to use `${CMAKE_BINARY_DIR}/spvgen` here (note: no CMAKE_**CURRENT**_BINARY_DIR, which refers to compiler/llpc/spvgen).

Looks good to me other than that.